src/Application/External/Context.elm

```diff
@@ -339,6 +353,7 @@ queryStringParser =
         |> apply (Query.custom "app" identity)
         |> apply (Query.custom "privatePath" identity)
         |> apply (Query.custom "publicPath" identity)
+        |> apply (Query.string "raw")
```

Comment:
Can you only have one raw ucan? Why not multiple?

Reply:
The request for raw ucans is a list of requests -- they are all base64 url encoded together.

Reply:
Oh ok! Alternatively you could add multiple query params with the same name, that's how the existing params work such as `publicPath`. A bit easier, but not required.